jib-plugins-common/src/test/java/com/google/cloud/tools/jib/plugins/common/SkaffoldFilesOutputTest.java

```diff
@@ -26,7 +26,7 @@
 public class SkaffoldFilesOutputTest {
```

Comment:
I find it strange that this is in plugins-common, and if it is shouldn't we be able to do an install and get a concrete version?

Reply:
About the plugins-common question, I think we decided to move it there in this PR https://github.com/GoogleContainerTools/jib/pull/1485#pullrequestreview-204434440

Reply:
This comment is a mistake, I thought this test was actually using the gradle/maven plugins and we were being lazy about it. I should've looked at the whole file.